components/Auction/createNewAuction.js

```diff
@@ -6,39 +6,70 @@ import fetchSelfDetails from '../../utils/fetchSelfDetails';
 
 const BASE_API_URL = process.env.NEXT_PUBLIC_BASE_API_URL;
 const AUCTIONS_URL = `${BASE_API_URL}/auctions`;
+const WALLET_URL = `${BASE_API_URL}/wallet`;
 
 const CreateNewAuction = () => {
   const router = useRouter();
 
-  const [itemType, setItemType] = useState();
+  const [itemType, setItemType] = useState('neelam');
   const [quantity, setQuantity] = useState();
   const [initialPrice, setInitialPrice] = useState();
   const [duration, setDuration] = useState();
   const [isSubmitted, setIsSubmitted] = useState(false);
+  const [userMoney, setUserMoney] = useState();
 
   useEffect(() => {
```

Comment:
If both useEffect do not have any dependencies, then you can consider having one useEffect.  
  fetchSelfDetails() &   getUserWallet() can stay in one useEffect.

Reply:
Done